### PR TITLE
Use element identifier when building report

### PR DESCRIPTION
**Pull request by mvdbeek**
That should help building a multiqc report.

Ping @nekrut

### CODE REVIEW

File: tools/fastqc/rgFastQC.xml [hunk 1]
```diff
@@ -1,22 +1,16 @@
...
+        python '$__tool_directory__/rgFastQC.py'
             -i '$input_file'
-            -d '${html_file.files_path}'
+            -d '$html_file.files_path'
             -o '$html_file'
             -t '$text_file'
-            -f '${input_file.ext}'
-            -j '${input_file.name}'
+            -f '$input_file.ext'
+            -j '$input_file.element_identifier'
```
Comment (jmchilton) [inline tools/fastqc/rgFastQC.xml]: Can you escape this 😇 ?

Comment (bgruening) [inline tools/fastqc/rgFastQC.xml]: @jmchilton should we add a `.element_identifier_escaped` in Galaxy to make this easier, or escape the default one and offer a `_plain` option?

**Comment by bgruening** [comment]
Thanks @mvdbeek!

File: tools/fastqc/rgFastQC.xml (hunk 1 above)
Comment (jmchilton) [inline tools/fastqc/rgFastQC.xml]: Yes I agree... we've talked about that before right?

**Pull request merged**